conda_build/noarch_python.py

```diff
@@ -70,13 +80,16 @@ def handle_file(f, d):
         d['site-packages'].append(g[14:])
 
     # Treat scripts specially with the logic from above
-    elif f.startswith('bin/'):
+    elif f.startswith(('bin/', 'Scripts')):
+        if f.endswith('.exe'):
+            os.unlink(path)  # we use the xx-script.py
+            return
         fn = basename(path)
-        rewrite_script(fn)
+        fn = rewrite_script(fn)
         d['python-scripts'].append(fn)
 
     # Include examples in the metadata doc
-    elif f.startswith('Examples/'):
+    elif os.path.split(f)[0] == 'Examples':
```

Comment:
In case the f is `Examples/foo/bar.py`, dirname would give `Examples/foo`, so we need split in this case.